### DISCUSSION TITLE
Wallet Sync? Way to know what's the height of the wallet chain?

### DISCUSSION THREAD

**Discussion by MasterJames** [Chia-Network/chia-blockchain#7657] [Q&A] [answered]
Well I am stuck syncing the wallet after it was stuck the first time and I deleted the dot folder it was in from other discussions, so it started rebuilding it but eventually has become stuck for days again. 

**I am asking how to know what the current height is suppose to be if it's synced?**
Is there a page online or something somewhere to show current heights etc.?

On Win10 GUI interface: wallet status is 460,032 and stuck for a few days now saying "Not Synced". [ as of 2021:07:19 ]

I had managed to switch to pooling and started rebuilding plots from Scratch so I had a few faucet credits and now I still see nothing like day one. 0 XCH 
_[It's of course a super disappointing user experience.]_

Restarting has resolved the jam, and Syncing has continued. Still there is no way to understand how long it will be until it's synced fully.
Restarting is annoying and not possible when you're plotting. If plotting was recoverable like in my crash at 97% 
https://github.com/Chia-Network/chia-blockchain/issues/7664
Then you'd be okay to restart to get the wallet sycing again? Alas, not crashing and not permanently freezing the wallet sync are better solutions but it would be nice to have recoverability.

**Comment by SlugFiller** [comment, marked as answer]
If you're running Windows GUI, chances are you're running both the wallet and full node. The full node has a display of the expected height while syncing. If it's not syncing, chances are you should restart the client, connect to additional nodes, and/or upgrade the client or check the internet connection.

The wallet height should be the same as the full node height. They are running on the same chain.

Also, you don't need to be synced in order to plot, and if you have the NFT address for plotting, you might as well do it. You can also run the plotter via command line (on Windows, the client installs to ~\AppData\Local\chia-blockchain\app-1.2.0\resources\app.asar.unpacked\daemon) and don't not have to worry about the GUI restarting. You can also use the MadMax plotter. It's also possible to plot without a net connection at all, so you can do it on a separate box, preparing your drives in advance, instead of simultaneously running a wallet and plotter on the same machine.

If you don't have the NFT address, however, you would most likely need to be synced in order to create it. At the very least, you would need confirmation and address of the mojo needed to be burned to create the NFT.

**Reply by MasterJames** [reply]
Right okay so the height is the same as the main chain. It's now almost about 600,100 and after restarting the GUI 12 hours later because it Jammed again it's carrying on to 599.321 in the wallet. Some XCH is showing again now.

I think this might be a memory leak or something like that? How would I check the logs for evidence of how it freezes?
Dev Tools only shows in the console SourceMap app.asar.build/enderer/js/493.js.map and main.js.map but those are just warnings.

**Reply by SlugFiller** [reply]
What you want to look at is $CHIA_ROOT/log/debug.log. The default value for $CHIA_ROOT is ~/.chia/mainnet. Be aware that even in a normally functioning client, the log can grow to around ~20mb per day, so it may be somewhat difficult to narrow down the cause of the problem.

Is the syncing only stopping for the wallet, or for the full node as well? From what I understand, the wallet is, by default, set to sync directly from the full node running on the same machine. So if the full node is out of sync, the wallet would be too.

If the full node goes out of sync, it could be down to the connected peers. Try using the "Connect to other peers" button, with one of the official reference peers.

**Reply by MasterJames** [reply]
Thanks I clearly should have dug around more. It's finally synced up again at and current at height 600,732. I will be able to better try and report the issue with the log later if and when it jams again. I am also going to try reloading from View Menu to see if that jump starts without restarting and interrupting plotting.
I think it was synced but the wallet wasn't. I understood from some other discussion threads, it's a separate chain but I guess it has the same height. I will try those ideas moving forward, thanks for your help and time.

**Comment by rmica** [comment]
Any news on #7171 ?

**Comment by MasterJames** [comment]
BTW: It jams frequently.
